antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -1072,7 +1072,9 @@ def clear_all_snapshots(self, retention_hours: timedelta, params: t.Optional[Req
         if params is None or (params.user and not params.user.is_site_admin() and not params.user.is_admin_token()):
             raise UserHasNotPermissionError()
 
-        task_name = f"Cleaning all snapshot updated or accessed at least {retention_hours} hours ago."
+        task_name = (
+            f"Cleaning all snapshot updated or accessed at least {retention_hours.total_seconds() // 3600} hours ago."
```

Comment:
We could use [humanize](https://pypi.org/project/humanize) so that it's even more readable for high values.
For example, when we have several days it will be better to have it in days than in hours.

For example:
```python
print(humanize.time.precisedelta(datetime.timedelta(days=12, hours=5)))
# --> 12 days and 5 hours
```

Also, we should rename the variable just retention_time because it's not necessarily hours.